### ISSUE TITLE
Do not use sketchy path determination

### ISSUE THREAD

**Issue by ethindp** [chrys87/fenrir#59]
Fenrir currently hard-codes its path and makes various hard assumptions about the Linux system it runs on. It assumes, for example, that the configuration directory is either `/etc/fenrirscreenreader` or `fenrir_binary_location/../../config`. This doesn't work on distributions like NixOS, where it is (not) best practice to store configuration under /etc (as that is used for global system configuration, and package-based configuration usually resides in the nix store to be manipulated with configuration options within configuration.nix). It is therefore highly advisable to provide a command-line option to override this determination and to specify the settings root.
